schedule/yam/agama_non_default_root_filesystem.yaml

```diff
@@ -11,3 +11,4 @@ schedule:
   - console/validate_blockdevices
   - console/validate_free_space
   - console/validate_read_write
+  - yam/validate/validate_connectivity
```

Comment:
add this validation module to schedules for default interactive installation and unattended one, as you designed the module to check connectivity in both ways.